bundles/org.openhab.binding.modbus.studer/src/main/resources/ESH-INF/thing/BSP-channel-types.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Power Channel Type -->
+	<channel-type id="Pbat">
+		<item-type>Number:Power</item-type>
+		<label>Power</label>
+		<state readOnly="true" pattern="%.2f %unit%" />
```

Comment:
Is the accuracy of all these values so high, that two digits after the decimal point are reasonable?

Reply:
For more accurate actions yes. Use remains at the user's discretion